playbooks/roles/mgmt_switch/templates/nexus.j2

```diff
@@ -1,25 +1,37 @@
 !
-hostname {{ device.hostname }}
+hostname {{ inventory_hostname }}
 !
-{% for iface in device.interfaces %}
-interface {{ device.interface_name }}/{{ iface.name }}
-{% if iface.type == "trunk" %}
-  switchport
-  switchport mode {{ iface.type }}
-  switchport {{ iface.type }} allowed vlan {{ iface.vlan }}
-{% elif iface.type == "access" %}
-  switchport
-  switchport mode {{ iface.type }}
-  switchport {{ iface.type }} vlan {{ iface.vlan }}
-{% else %}
-  no switchport
+{% for vlan in vlans %}
+{% if "N9K" in vlan.notes %}
```

Comment:
I would remove this 'IF'. All the lab VLANs need to be configured on the N9Ks, the "notes" fields I'd rather not use as they are not reliable

Reply:
if's is the case, agree! it makes life easier